.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
```

Comment:
- [ ] Please fix the background position to match the design
![image](https://user-images.githubusercontent.com/74744787/185410345-97780e67-3ffd-4811-bf34-5706e959e82b.png)

Reply:
i already fixed the codes you are showing was for the first pull request